### PR TITLE
add value on PR details

### PR DESCRIPTION
**Pull request by brunoparma88**
Add hours to each PR.

image

### CODE REVIEW

File: spec/requests/pull_requests/time_to_merge_prs/index_spec.rb [hunk 1]
```diff
@@ -89,7 +89,7 @@
 
     it 'returns data only for all repositories' do
       assigns(:pull_requests).each do |_, pull_request_url|
-        expect(all_pull_requests_url).to include(pull_request_url.first)
+        expect(all_pull_requests_url).to include(pull_request_url.first[:html_url])
```
Comment (megatux) [inline spec/requests/pull_requests/time_to_merge_prs/index_spec.rb:92]: what about validating the new _value_ added?

**Review by megatux — COMMENTED** [review]
LGTM. Added a comment on tests

File: app/assets/stylesheets/application.scss [hunk 2]
```diff
@@ -143,6 +143,10 @@ main {
   background-color: #d2f4ea;
 }
 
+.hours-pr{
+  color: grey
```
Comment (megatux) [inline app/assets/stylesheets/application.scss:147]: missing semicolon

File: spec/requests/pull_requests/time_to_merge_prs/index_spec.rb (hunk 1 above)
Comment (brunoparma88) [reply spec/requests/pull_requests/time_to_merge_prs/index_spec.rb:92]: The value already existed. It is validated in "Events PullRequest Spec"

File: yarn.lock [hunk 3]
```diff
@@ -1746,9 +1746,9 @@ caniuse-api@^3.0.0:
     lodash.uniq "^4.5.0"
 
 caniuse-lite@^1.0.0, caniuse-lite@^1.0.30000981, caniuse-lite@^1.0.30001043, caniuse-lite@^1.0.30001061:
```
Comment (enzofab91) [inline yarn.lock:1748]: I think this file is not from this PR. WDYT?

Comment (brunoparma88) [reply yarn.lock]: I separate it for another PR

**Review by santib — APPROVED** [review]
I'm not sure why we don't use feature/system specs and check that the correct texts are being rendered, but 🤷‍♂️

**Pull request merged**